src/arrayMethodSort.js

```diff
@@ -1,11 +1,44 @@
-'use strict';
+"use strict";
 
 /**
  * Implement method Sort
+ *
+ *
  */
+
+function compareStrings(a, b) {
+  const stringA = String(a);
+  const stringB = String(b);
+
+  if (stringA > stringB) {
+    return 1;
+  } else if (stringA === stringB) {
+    return 0;
+  } else {
+    return -1;
+  }
```

Comment:
when you use `if` statements and return something, you can use next syntax

```
if (a > b) {
  return x
}

if (a < b) {
  return y
}

return c
```

Reply:
Thanks!